regimes/in/extensions.go

```diff
@@ -0,0 +1,30 @@
+package in
+
+import (
+	"github.com/invopop/gobl/cbc"
+	"github.com/invopop/gobl/i18n"
+	"github.com/invopop/gobl/pkg/here"
+)
+
+// Extension keys used in India.
+const (
+	ExtKeySupplyPlace cbc.Key = "in-supply-place"
```

Comment:
If an invoice is not issued under GST, meaning the transaction is outside the purview of GST (e.g., exempted goods/services, non-GST supplies, or transactions by unregistered individuals/entities not liable for GST), then Place of Supply is not required to be mentioned on the invoice. This is because the tax determination that relies on the Place of Supply does not apply in such cases.

https://cleartax.in/s/place-of-supply-of-goods